.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!---
+Please write your PR name in the present imperative tense. Examples of present imperative tense are:
+"Fix issue in the dispatcher where…", "Improve our handling of…", etc."
+
+For more information on Pull Requests, you can reference here:
+https://success.vanillaforums.com/kb/articles/228-using-pull-requests-to-contribute
+-->
+## Describe your changes
+
+
+## Non-obvious technical information
+
+
+## Checklist before requesting a review
+- [x] pre-commit run --all-files (run before pushing)
+- [x] pytest if applicable
+- [x] Link issue
+- [x] Update relevant documentation if applicable: doc strings, readme, poetry.
```

Comment:
@aaronhaefner, for the template we'll want to make sure they are unchecked. Otherwise, LGTM!